myTrip/tests/unittests/registration/test_models.py

```diff
@@ -0,0 +1,104 @@
+"""Contains model tests for registration app."""
+
+from datetime import datetime
+from django.test import TestCase
+
+from registration.models import CustomUser
+
+
+class CustomUserModelTests(TestCase):
+    """
+    Test CustomUser model methods.
+    """
+
+    def setUp(self):
+        """
+        Create CustomUser record in database.
+        """
+
+        date = datetime.now()
+        self.user = CustomUser.objects.create(id=3,
+                                              first_name='Roman',
+                                              last_name='Hrytskiv',
+                                              email='test@gmail.com',
+                                              created=date,
```

Comment:
for some reason, when i tried to add date in next few lines with
self.user.created ...
it throwed an error about nonull constraint blah blah, all test failed, so i had to redo this